src/test/java/com/artipie/rpm/RpmTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm;
+
+import com.artipie.asto.Concatenation;
+import com.artipie.asto.Content;
+import com.artipie.asto.Key;
+import com.artipie.asto.Storage;
+import com.artipie.asto.memory.InMemoryStorage;
+import java.nio.charset.Charset;
+import java.nio.file.Path;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * Unit tests for {@link Rpm}.
+ *
+ * @since 0.9
+ * @todo #63:30min Don't change metadata when invalid package is sent.
+ *  Currently Rpm is recalculating metadata when an invalid package is sent.
+ *  It should not. Correct that and enable the test below.
+ */
+final class RpmTest {
+
+    /**
+     * Path of repomd.xml fil.
+     */
+    private static final String REPOMD = "repodata/repomd.xml";
+
+    @Test
+    @Disabled
+    void doesntBrakeMetadataWhenInvalidPackageSent(@TempDir final Path tmp)
+        throws Exception {
+        final Storage storage = new InMemoryStorage();
+        final Rpm repo =  new Rpm(
+            storage, StandardNamingPolicy.SHA1, Digest.SHA256, true
+        );
+        storage.save(
+            new Key.From("repo/oldfile.txt"),
```

Comment:
@paulodamaso rpm metadata generator will ignore this file: it's located in subdirectory, but rpm repository stores rpm files in the repo root, it doesn't have `.rpm` extension.

Reply:
@g4s8 I didn't understood the comment very well, what should I do to rpm data generator NOT ignore this files?

Reply:
@paulodamaso see the source of `Rpm.batchUpdate()`:
```java
.filter(key -> key.string().endsWith(".rpm"))
```
it includes only `.rpm` files in **root** of sepcified directory, it means that `Rpm` ignores all `.txt` packages and packages in `repo/` directory